src/components/filter/FilterPanel.tsx

```diff
@@ -124,9 +122,6 @@ const FilterPanel = (props: Props) => {
 			<div className={filterButtonClass} ref={filterButtonRef} onClick={onClick}>
 				<div className={filterIconClass} />
 				<div className={filterTitleClass}>{showFilter ? 'Hide Filter' : 'Show Filter'}</div>
-				{typeof isLoading === 'boolean'
```

Comment:
I see at least one scenario when having loading indicator is good.
If messages list contains messages, then you select some event, and while messages are loading(and they can all really long we had this problem) you should show some indicator that something is loading, right now if attached messages are loading stop loading button is active which is not right aswell

Reply:
While we loading attached messages we reset the messages list and have big spinner at the center of empty message list

Reply:
No, we don't. Plus if filter is applied it will filter all attached messages, and they don't match current filter message list will not update